src/docs/product/session-replay/getting-started.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Set Up
+sidebar_order: 1
+redirect_from:
+  - /session-replay/setup/
+  - /session-replay/getting-started/
+description: "Get started with Session Replay, TODO."
+---
```

Comment:
Child page in the Product section:

<img width="291" alt="SCR-20221205-nrc" src="https://user-images.githubusercontent.com/187460/205780580-8d82d2d7-4ce5-451e-b92c-58a64cf92dff.png">

Reply:
I imagine Set Up is mainly a client-side/SDK thing? If so, we could probably get by without the sub page to start? Basically add all content on the landing Session Replay page for now

Reply:
Agree that Set Up should in the SDK docs ^ Not in the Product Guide section

Reply:
Ah just took a look at the "Performance Monitoring" section again, it looks like Ryan is following the pattern of what's done there. The "Set Up" just includes a list of links to the SDK docs and that makes sense to me. I wouldn't want to clutter product docs w/ installation instructions

Reply:
yup! copy+pasted from Perf

Reply:
I can pull these Product pages out of this PR and do them in a separate PR

Based on https://github.com/getsentry/sentry-docs/issues/5903 we don't have a specific product pages drafted yet (maybe faq?)